client/src/cmdusart.c

```diff
@@ -392,7 +415,7 @@ static int CmdUsartBtPin(const char *Cmd) {
     }
 
     char string[6 + sizeof(pin)] = {0};
-    snprintf(string, sizeof(string), "AT+PIN%s", pin);
+    snprintf(string, sizeof(string), "AT+PIN=%s", pin);
```

Comment:
I wonder, 
yours has a  `=`  in `AT+PIN=%s`
and the old one doesn't.
`AT+PIN%s` 

That feels like a breaking change.